KinSDK/KinSDK/Core/KinAccount.swift

```diff
@@ -362,25 +357,53 @@ final class KinStellarAccount: KinAccount {
             watch = nil
 
             linkBag = LinkBag()
-            
+
             p.signal(())
         }).add(to: linkBag)
 
         return p
     }
 
+    /**
+     Exports this account as a Key Store JSON string, to be backed up by the user.
+
+     - parameter passphrase: The passphrase used to create the associated account.
+     - parameter exportPassphrase: A new passphrase, to encrypt the Key Store JSON.
+
+     - throws: If the passphrase is invalid, or if exporting the associated account fails.
+
+     - returns: a prettified JSON string of the `account` exported; `nil` if `account` is `nil`.
+     */
     @available(*, unavailable)
     private func exportKeyStore(passphrase: String, exportPassphrase: String) throws -> String? {
         let accountData = KeyStore.exportAccount(account: stellarAccount, passphrase: passphrase, newPassphrase: exportPassphrase)
-        
+
         guard let store = accountData else {
             throw KinError.internalInconsistency
         }
-        
+
         guard let jsonData = try? JSONSerialization.data(withJSONObject: store, options: [.prettyPrinted]) else {
             return nil
         }
-        
+
         return String(data: jsonData, encoding: .utf8)
     }
 }
+
+// MARK: - Deprecated
```

Comment:
so we're deprecating the old methods and not yet deleting them?
@drEfrati let's consider doing this in Andorid as well, that's a small effort for being more sentiment to our devs.